package.json

```diff
@@ -111,7 +111,7 @@
   },
   "license": "SEE LICENSE.md",
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=18.0.0"
```

Comment:
Should this be `>=18.18.0`?

Reply:
anything over 18 will work. I wanted to keep it flexible but I can update